source-code/website/src/pages/Layout.tsx

```diff
@@ -80,6 +80,12 @@ function Header(props: { landingpage?: boolean }) {
 	const links = [
 		{ name: "Blog", href: "/blog", type: "text" as buttonType },
 		{ name: "Docs", href: "/documentation", type: "text" as buttonType },
+		{
+			name: "Share Feedback",
+			external: true,
+			href: "https://github.com/inlang/inlang/discussions/categories/feedback",
+			type: "textPrimary" as buttonType,
```

Comment:
Wow, thank you!

why did you choose "primary" over "text" like the other buttons?

Reply:
Hi @boian-ivanov, thank you, that's great. 
I would go for the `text`. So it's consistent with the other links.
I think it's really important to have this button in the Header, but I think on the marketing page it looks a bit cluttered. 
Maybe we could fix it by adding this condition:
`external: currentPageContext.urlParsed.pathname.includes("editor"),` in the object. 
This way we have the nice affordance in the editor but no cluttered header on the marketing page.

Reply:
@NilsJacobsen I tend to agree that it's cluttered with the text being too large, so I think shortening it to just "Feedback" should be fine. My only issue with removing the chevron (even for some of the pages) is that it wouldn't show any indication that it's an external link. Ideally instead of a chevron, a symbol for an external link would be used (like the one bellow). 

<img src="https://user-images.githubusercontent.com/15968453/228487106-31f5cee4-2f33-4145-b148-0767873aaf86.png" width="20" />

But still, I'm quite open to any ideas and if you guys feel that it won't need one, I wouldn't mind making the change.